R/id_outbreak_peak.R

```diff
@@ -0,0 +1,14 @@
+# Function to estimate the peak of the outbreak using incidence2
+# Incidence object from linelist has 3 count_variables, but we want the column
+# date_onset only to be considered to estimate the peak
+id_outbreak_peak <- function(linelist) {
+  inc_data <- linelist %>%
```

Comment:
This function remakes an `<incidence>` object, but one is already created in the analysis script before calling this function. Might be worth seeing the `<incidence>` already made can be passed into this function and save yourself a step.

Reply:
This is what I meant the other day about having problems with this step, the thing is that in order to be able to use incidence2's function to estimate the outbreak peak, you need to start off an incidence object, and then it takes the date_index column to give you the peak. The problem is that when there are more than 1 elements in date_index, it calculates the peak for each of them, not just for the onset date (in my case, also for the deaths and recoveries). I tried starting off the previous incidence object that I use in the analyses and remove the rows in date_index that said "date_death" and "date_recovery" first so it would only take into account onset dates, but I didn't find it easy to do at the time and I was getting some unexpected results, so I decided it was best to start off the "correct" incidence object
If there is an easier way to do this I'm more than happy to change it